src/super_gradients/training/models/classification_models/efficientnet.py

```diff
@@ -599,70 +609,200 @@ def get_efficientnet_params(width: float, depth: float, res: float, dropout: flo
 class EfficientNetB0(EfficientNet):
     def __init__(self, arch_params):
         blocks_args, arch_params = get_efficientnet_params(width=1.0, depth=1.0, res=224, dropout=0.2, arch_params=arch_params)
-        super().__init__(blocks_args=blocks_args, arch_params=arch_params)
+        super().__init__(
+            blocks_args=blocks_args,
+            num_classes=arch_params.num_classes,
+            backbone_mode=arch_params.backbone_mode,
+            batch_norm_momentum=arch_params.batch_norm_momentum,
+            batch_norm_epsilon=arch_params.batch_norm_epsilon,
+            image_size=arch_params.image_size,
+            width_coefficient=arch_params.width_coefficient,
+            depth_divisor=arch_params.depth_divisor,
+            min_depth=arch_params.min_depth,
+            depth_coefficient=arch_params.depth_coefficient,
+            dropout_rate=arch_params.dropout_rate,
+            drop_connect_rate=arch_params.drop_connect_rate,
+        )
 
 
 @register_model(Models.EFFICIENTNET_B1)
 class EfficientNetB1(EfficientNet):
     def __init__(self, arch_params):
```

Comment:
will you add typing to all these __init__ in the next phase of the documentation fix?

Reply:
I added typing to arch_params, is it what you meant?93c77020d1a5cf43cd2a75f2acf506d842bf4036